storybook/stories/Examples/RadarChart.stories.tsx

```diff
@@ -91,7 +91,7 @@ export const RadarWithLegend: StoryObj = {
       <RadarChart data={data} width={360} height={360}>
         <PolarGrid gridType="circle" />
         <Legend />
-        <Tooltip />
+        <Tooltip defaultIndex={2} />
```

Comment:
The change seems unrelated to the Story `RadarWithLegend`. Should we possibly have a separate story for the Tooltip with the Radar Chart, which then has defaultIndex set? 

Or rather, should these examples with the Tooltip having non default values not be done on the Tooltip stories instead? 

I have no clear preference, but I think it would be great for documentation readers to always be only exposed to a single concept / component per story.

For the Tooltip defaultIndex, I suggest to change the Tooltip stories. Does that make sense?

Reply:
I guess we have to strike a balance here.

I think @PavelVanecek is trying to get stories in place that will break in snapshots if we do something that breaks the feature. That makes sense to me from a development perspective. It doesn't always make sense from a documentation perspective. 

I wish there was some way we could get the visual diffs without exposing the stories themselves.

Reply:
I am after the visual diff.